Library/Homebrew/github_packages.rb

```diff
@@ -263,7 +263,9 @@ def upload_bottle(user, token, skopeo, formula_full_name, bottle_hash, keep_old:
     root.mkpath
 
     if keep_old
-      download(user, token, skopeo, image_uri, root, dry_run: dry_run)
+      # Always pass false to dry_run because we'll need this metadata
+      # for later parts of the keep_old workflow to succeed.
+      download(user, token, skopeo, image_uri, root, dry_run: false)
```

Comment:
This feels a bit weird to me. Could it pass `keep_old` instead and adjust the `download` behaviour accordingly?

Reply:
As in push the logic down into `download` rather than dealing with it here?

Reply:
@mistydemeo Yeh, that'd be great, thanks!